spec/lib/bamboozled/employee_spec.rb

```diff
@@ -100,6 +100,35 @@
     url.must_equal required_url
   end
 
+  it 'creates a new employee in BambooHR' do
```

Comment:
@Enriikke to confirm you want something like:

`
 describe "#title" do
    it "returns the title of the blog" do
      @blog.title.must_equal "Some Title"
    end
  end
`

Personally I would like to change all the tests in employee_spec.rb to this style as I find it more clear.